pkgs/tools/system/gatotray/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, pkg-config, gdk-pixbuf, gtk2, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gatotray";
+  version = "3.3";
+
+  src = fetchFromGitHub {
+    owner = "gatopeich";
+    repo = "gatotray";
+    rev = "v${version}";
+    sha256 = "08hy1zyq0nkrhmp599x6k0sn93a4484yi2lfgl0xf25lwdsrsyr5";
+  };
+
+  # Fix https://github.com/gatopeich/gatotray/issues/4 for latest stable version
+  patches = [ ./0001-Fix-issue-4-free-invalid-pointer.patch ];
+
+  postPatch = ''
+    # Substitute hard-coded install destinations
+    substituteInPlace Makefile \
+      --replace /usr/local/bin $out/bin \
+      --replace /usr/share/ $out/share/
+
+    # Leave stripping to fixup phase
+    sed -i "/strip/d" Makefile
```

Comment:
I think many packages do this on their own. I don't think we should patch it, it doesn't change anything.